pre-commit.sh

```diff
@@ -1,5 +1,10 @@
 #!/bin/sh 
 
+git_dir=$(git rev-parse --git-dir)
+if [ -f "$git_dir/hooks/pre-commit" ]; then
+    "$git_dir/hooks/pre-commit" "$@"
+fi
```

Comment:
Do I understand right that execution of this hook will continue if `$git_dir/hooks/pre-commit` exits 0? 

if `$git_dir/hooks/pre-commit` exits non-zero, should use use `set -e` so this script also fails?

Reply:
I suppose we could `set -e` before running `$git_dir/hooks/pre-commit` and `set +e` after